autogen/docs/programming.html.mustache

```diff
@@ -2871,7 +2871,14 @@ ifelse patch-at (new-x - xcor) (new-y - ycor) = nobody
       if your anonymous procedure takes no arguments.
       For example one may write simply <code>foreach mylist [ print ]</code>
       instead of <code>foreach mylist [ [x] -&gt; print x ]</code>, though the latter
-      is also accepted.
+      is also accepted. Additionally, while brackets around argument names are required
+      for anonymous procedures of two or more arguments, they may be omitted for
+      anonymous procedures of one or zero arguments. So <code>foreach mylist [ [x] -&gt; print x ]</code>
+      is equivalent to <code>foreach mylist [ x -&gt; print x ]</code>.
```

Comment:
I'd flip the two versions, i.e.:

    So <code>foreach mylist [ x -&gt; print x ]</code> is equivalent to <code>foreach mylist [ [x] -&gt; print x ]</code>.